Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>nit-picky comment - more for future, you don't have to change here. Python die-hards always use lowercase for identifiers like func names - except classes (camel-cased) or constants (upper case). Word separation is done with underscores. - view_pca vs, viewPCA.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>